src/js/lib/Scrollspy.jsx

```diff
@@ -68,7 +70,8 @@ export class Scrollspy extends React.Component {
       }
 
       const isLastItem = i === max - 1
-      const isScrolled = (document.documentElement.scrollTop || document.body.scrollTop) > 0
```

Comment:
`document.body.scrollTop` appears to be deprecated and I'm not sure if it's something we should continue to support. I guess we could add this to the rootNode accessor for backwards compatibility.